src/plugin.js

```diff
@@ -594,7 +631,7 @@ class VR extends Plugin {
     }
 
     // reset the video element style so that it will be displayed
-    this.getVideoEl_().style.display = '';
+    this.getVideoEl_().style.opacity = '';
```

Comment:
I assume this is from your other pull request, which is now in master. Feel free to rebase and this should go away.